geolink_formatter/utils.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+""" Provides additional methods usable for parsing the xml document from the geolink api. """
+
+
+def filter_duplicated_documents(documents):
```

Comment:
I'm wondering whether the geolink_formatter should do this.
It's more scope than what it's currently doing, and I think there is also code in pyramid_oereb to remove duplicates.

Reply:
Good point. The functionality is not new (https://github.com/openoereb/geolink_formatter/blob/a27645c5c7c692e50cfb0d39e0fda4cd2cedc7ec/geolink_formatter/parser.py#L123) in this PR but we could remove it. I wonder if having two documents with the same id in a lexlink is possible at all.

Reply:
I think this is necessary as we use the HTML rendering of the documents in our WebGIS. Duplicate documents should not occur but might be possible, e. g. if a hint docuement is attached twice (to law and legal provision).